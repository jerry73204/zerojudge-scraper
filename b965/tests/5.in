8 3 6
2 3 8
8 5 8
5 2 0
3 7 6
2 5 1
7 6 3
0 3 2
1 5 6
1 1 1 0 0 0