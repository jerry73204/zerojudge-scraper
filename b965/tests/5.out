3 8
6 2 3 1 6 0 8 8
5 3 6 5 7 2 5 3
1 0 7 2 3 5 8 2